In [10]:
import numpy as np
from scipy.stats import binom

In [25]:
prob = binom.cdf(50, 100, 0.5) + 1 - binom.cdf(60.5, 100, 0.5)
print(prob)

0.5573947188024422


Probability that in 100 coin flips, the number of head I get is between 50 and 60 is the .56